### DISCUSSION TITLE
Disable animations?

### DISCUSSION THREAD

**Discussion by damienzhang** [vincode-io/Zavala#158] [Ideas]
Thanks for building this amazing app and giving it away for free. I'm very interested in using it to replace Workflowy, and one thing I've noticed is that Zavala feels a little less snappy due to the animation when a new item is added to the outline. Workflowy creates a new bullet instantly, and it's very satisfying. Is it possible to have a switch in Zavala to disable or speed up animations? Bike, for example, has such a setting.

**Comment by vincode-io** [comment]
I've created a GitHub issue to keep track of this feature request for future consideration.  #159
